LearningHub.Nhs.WebUI/Controllers/Api/ContributeController.cs

```diff
@@ -465,6 +465,10 @@ public async Task<ActionResult> SaveResourceDetailAsync([FromBody] ResourceDetai
             request.Description = Regex.Replace(request.Description, "<p> ", "<p>");
             request.Description = Regex.Replace(request.Description, "<p></p>", string.Empty);
             request.Description = Regex.Replace(request.Description, "\\n", string.Empty);
+            if (request.NodeId > 0)
```

Comment:
The request.NodeId here is only set to the Catalogue NodeId if the resource is contributed to hte root of the catalogue.
If the resource is contributed directly to a folder within a catalogue this is set to the Folder NodeId & not the Catalogue NodeId
<img width="886" alt="image" src="https://github.com/user-attachments/assets/a50a2097-e555-4e25-bf04-884821629b4a">